pkg/frontend/querymiddleware/roundtrip.go

```diff
@@ -411,14 +420,17 @@ func IsInstantQuery(path string) bool {
 
 func IsCardinalityQuery(path string) bool {
 	return strings.HasSuffix(path, cardinalityLabelNamesPathSuffix) ||
-		strings.HasSuffix(path, cardinalityLabelValuesPathSuffix) ||
-		strings.HasSuffix(path, cardinalityActiveSeriesPathSuffix)
+		strings.HasSuffix(path, cardinalityLabelValuesPathSuffix)
 }
```

Comment:
One of the side-effects here is that we no longer apply the `newCardinalityQueryCacheRoundTripper` middleware to active series endpoint, is that intended?

Reply:
Yes, that's intended, I've tried to explain the reasoning in the PR description.

> This PR also introduces a dedicated roundtripper for active series requests to bypass the generic cache. Since this endpoint is supposed to return "fresh" data, setups with a large cache TTL config could yield outdated results if the cache is enabled and not manually bypassed for querying.